gobblin-modules/gobblin-azkaban/src/main/java/org/apache/gobblin/azkaban/AzkabanGobblinYarnAppLauncher.java

```diff
@@ -135,10 +135,6 @@ public void run() {
 
   @Override
   public void cancel() throws Exception {
-    try {
-      this.gobblinYarnAppLauncher.stop();
-    } finally {
-      super.cancel();
```

Comment:
This is not necessary because super.cancel throws a not implemented exception, causing clutter in the logging